plugins/services/src/js/containers/tasks/TasksView.js

```diff
@@ -141,25 +142,50 @@ class TasksView extends mixin(SaveStateMixin) {
     );
   }
 
-  getKillButtons(hasCheckedTasks) {
-    if (!hasCheckedTasks) {
+  getKillButtons() {
+    const {tasks} = this.props;
+    const {checkedItems} = this.state;
+
+    if (!Object.keys(checkedItems).length) {
       return null;
     }
+    // Only show Stop if a scheduler task isn't selected
+    const hasSchedulerTask = tasks
+      .some((task) => task.id in checkedItems && task.schedulerTask);
+
+    // Using Button's native "disabled" prop prevents onMouseLeave from
+    // being correctly called, preventing correct dismissal of the Tooltip.
+    //
+    // So we manually disable the button.
+    let onStopClick = () => {};
```

Comment:
I'd use a good old `function` here, as the arrow functiion looks a little funky...  #nit